content/React Query and management of server state.md

```diff
@@ -0,0 +1,75 @@
+# React Query and management of server state
+
+Some days ago the [React Summit Remote Edition](https://remote.reactsummit.com/) gathered tons of people behind the screen with a promise of put together popular speakers around the React Community like [Kent C. Dodds](https://kentcdodds.com/) and [Guillermo Rauch](https://rauchg.com/) among others, but one talk especially caught my attention, *React Query: It’s Time to Break up with your "Global State”!* by [Tanner Linsley](https://twitter.com/tannerlinsley) were joining two pain points that I had frequently with React, such as how to handle asynchronous information in the global state and how to restructure my code to be less repetitive and not force async data to follow synchronous flow.
```

Comment:
I would put a period after Tanner Linsley and continue the rest as a new sentence, like `In it Tanner talked about two pain points...`

Reply:
Agree, the sentence was too long, thanks